src/frontend/src/pages/CreditsPage/CreditsPage.tsx

```diff
@@ -124,18 +124,6 @@ const CreditsPage: React.FC = () => {
     { name: 'Aarav Shyamkumar', color: '#FF0000' },
     { name: 'Raghav Mathur', color: '#009933' },
     { name: 'Anika Sharma', color: '#ff0000' },
-    {
```

Comment:
please don't delete all these people's names

Reply:
Oh shoot I don’t know how that happened